src/styles/main.scss

```diff
@@ -20,6 +20,9 @@ body {
   text-align: center;
   vertical-align: center;
```

Comment:
The 'vertical-align' property doesn't have a 'center' value. If you're trying to center the text vertically, consider using 'display: flex' along with 'align-items: center' and 'justify-content: center' for the '.field-cell' class.